.github/workflows/ci.yml

```diff
@@ -1,6 +1,10 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
I'm 99% sure we had to have both because of forks

Reply:
We can use `pull_request_target` which is what I do on primer/css that will run on forks

Reply:
Example here https://github.com/primer/css/pull/1357